src/styles/main.scss

```diff
@@ -1,3 +1,209 @@
+$blue: #00acdc;
+$dark-blue: #060b35;
+$gray: #616070;
+$white: #fff;
+$font-family: "Roboto", sans-serif;
+
+@import "header";
+
+html {
+  color: $dark-blue;
+  font-size: 8px;
+  font-family: $font-family;
+}
+
 body {
   margin: 0;
 }
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+
+  display: flex;
+  align-items: center;
+
+  height: 60px;
+  font-family: $font-family;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: $dark-blue;
+
+  transition: color 0.3s;
+}
+
+.nav__link:hover {
+  color: $blue;
+}
+
+.nav__link.is-active {
+  color: $blue;
+}
+
+.nav__link.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+
+  height: 4px;
+  border-radius: 2px;
+  background-color: $blue;
+}
+
+.cards {
+  justify-content: center;
+  padding-top: 50px;
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+
+  gap: 46px 48px;
+}
+
+.card {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $white;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+}
+
+.card__img {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.product-name {
+  color: $dark-blue;
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.product-code {
+  font-size: 10px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.review {
+  margin-bottom: 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__active {
```

Comment:
.stars__star { 

and use modifier and loop from previous `layout__stars` task 
![image](https://github.com/mate-academy/layout_catalog/assets/70009205/7ea4885f-b346-474b-8b82-9c50aecfb5ad)